exercises/practice/sgf-parsing/.meta/config.json

```
@@ -0,0 +1,15 @@
+{
+  "authors": [],
```

```suggestion
  "authors": ["tlphat"],
```

Since you ported the exercise, you deserve to be the author of it!

---

Thanks, I've added myself as the author.